go.mod

```diff
@@ -65,8 +65,13 @@ require (
 // When updating cloud.google.com/go/auth, ensure the auth.IsCredentialsNotFoundError still works.
 require cloud.google.com/go/auth v0.12.0
 
+require cloud.google.com/go/logging v1.11.0
```

Comment:
It might be worth migrating to go workspaces, so the dependencies for this sub-component aren't added to the dependencies of the parent component.

Probably dont want to do that in this PR tho.